File: fibonacci_series/nim/fibonacci.nim
import strutils as str
echo "enter how many fibonacci numbers you want"

var num:int = str.parseInt(readLine(stdin))

var num0: int = 0
var num1: int = 1
var num2: int = 0 
var i: int= 1

while i < num or i == num:
    echo num1
    num0 = num1
    num1 = num2
    num2 = num0+num1
    i = i + 1

File: fibonacci_series/PHP/fibonacci.php
<?php  
/* Print fiboancci series upto 12 elements. */  
$num = 12;  

$input = readline("How many numbers to generate?");
if( $input ) $num = 1*$input; 

echo "<h3>Fibonacci series using recursive function:</h3>";  
echo "\n";
/* Recursive function for fibonacci series. This is fairly inefficient. */  
function series($num){
    if($num == 0){
        return 0;
    } else if( $num == 1) {
        return 1;
    } else {
        return (series($num-1) + series($num-2));  
    }
}
/* Call Function. */  
for ($i = 0; $i < $num; $i++){  
    echo series($i);  
    echo "\n";  
}  
